climada/util/coordinates.py

```diff
@@ -783,12 +783,22 @@ def get_country_geometries(
     if country_names:
         if isinstance(country_names, str):
             country_names = [country_names]
+
+        # print warning if ISO code not recognized
+        for country_name in country_names:
+            if not country_name in nat_earth[["ISO_A3", "WB_A3", "ADM0_A3"]].values:
+                LOGGER.warning(f"ISO code {country_name} not recognized.")
```

Comment:
wouldn't it be better to simply throw an exception in case the country is not recognized?
I can't see the point of filtering out the unrecognizable countries in here. If a user runs this with a typo in one of the country names would they be upset if it fails?

Reply:
Yes I agree, this would be better. I changed it to raise a ValueError now.

My only doubt is that the function util.coordinates.get_country_geometries() existed before and if you input a list of ISO codes with one invalid code, it did NOT raise an error before (this is why I "only" added a warning). Wouldn't this be a potential problem for existing code to break down due to the error, or is it better to make aware that the input was not correct?

Reply:
@emanuel-schmid What do you think?

Reply:
Good question. Imo, it is better to make them finally aware that their input has been wrong all the time.
We just need to add a note in the change log.

Reply:
Perfect, agreed. I'm adapting the change log accordingly.